app/controllers/application_controller.rb

```diff
@@ -1,5 +1,15 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  
   protect_from_forgery with: :exception
+
+  protected
+
+  def configure_permitted_parameters
+  	devise_parameter_sanitizer.for(:sign_up) << :first_name
```

Comment:
this method doesn't look dry. Please, consider this article about ruby array to find out how this method can be rewritten http://docs.ruby-lang.org/en/2.0.0/Array.html#method-i-3C-3C